psl-core/src/test/java/org/linqs/psl/application/inference/InferenceTest.java

```diff
@@ -278,28 +278,33 @@ public void testNoCommitAtom() {
     public void initialValueTest() {
         TestModel.ModelInformation info = TestModel.getModel();
 
-        double oldObjective = 0.0f;
+        double oldAvgObjective = 0.0f;
         InitialValue oldInitialValue = null;
 
         for (InitialValue initialValue : InitialValue.values()) {
             Options.INFERENCE_INITIAL_VARIABLE_VALUE.set(initialValue.toString());
 
             Set<StandardPredicate> toClose = new HashSet<StandardPredicate>();
-            Database inferDB = info.dataStore.getDatabase(info.targetPartition, toClose, info.observationPartition);
-            InferenceApplication inference = getInference(info.model.getRules(), inferDB);
 
-            double objective = inference.inference();
-            inference.close();
-            inferDB.close();
+            double avgObjective = 0.0;
+            for (int i = 0; i < 25; i++) {
```

Comment:
Magic number.

Averaging out multiple runs is a good idea, so other tests may want to use this constant.
But, 25 runs seems a bit high.
How much longer does it take?
What about 5-10 runs?